packages/manager/apps/dedicated/client/app/components/ip/agoraOrder/ipv6/ipv6.constant.js

```diff
@@ -0,0 +1,51 @@
+import { TRACKING_PREFIX, ALERT_ID } from '../ip-ip-agoraOrder.constant';
+
+export const PRODUCT_TYPES = {
+  vrack: {
+    apiTypeName: 'VRACK',
+    typeName: 'VRACK',
+  },
+};
+
+export const FLAGS = {
+  par: 'fr',
+  gra: 'fr',
+  sbg: 'fr',
+  lim: 'de',
+  waw: 'pl',
+  eri: 'gb',
+  vin: 'us',
+  hil: 'us',
+  bhs: 'ca',
+  sgp: 'sg',
+  syd: 'au',
+  rbx: 'fr',
+  tor: 'ca',
+  mum: 'in',
+  '1-preprod': 'fr',
+  '1-dev-2': 'fr',
+  '1-dev-1': 'fr',
+  bru: 'be',
+  mad: 'es',
+  dal: 'us',
+  lax: 'us',
+  chi: 'us',
+  nyc: 'us',
+  mia: 'us',
+  pao: 'us',
+  den: 'us',
+  atl: 'us',
+  mrs: 'fr',
+};
+
+export const EMPTY_CHOICE = '-';
+
+export { TRACKING_PREFIX, ALERT_ID };
```

Comment:
What's the goal here ?
It's not necessary as those constants are already exported in `ip-ip-agoraOrder.constant`

Reply:
rebase and conflict issue